bench/workgen/workgen.cpp

```diff
@@ -362,19 +383,27 @@ WorkloadRunner::start_tables_create(WT_CONNECTION *conn)
             continue;
         }
 
-        for (int creates = 0; creates < _workload->options.create_count;
-             ++creates, ++total_creates) {
-            char uri[BUF_SIZE];
-            snprintf(uri, BUF_SIZE, "table:%s%" PRIu64, _workload->options.create_prefix.c_str(),
-              total_creates);
+        int creates = 0;
+        while (creates < _workload->options.create_count) {
+            char rand_chars[DYNAMIC_TABLE_LEN];
```

Comment:
Could we use a `std::string` instead and then concatenate it with the `uri` defined below? This is more c++ and would avoid the handling of the null terminator.

Reply:
As mentioned in the previous comments, C-styled char array allocated on the stack better suit random table name generation. I could replace it with ```std::array```, but C styled strings are more intuitive.